src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_TIME;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ScheduleCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Meeting;
+
+/**
+ * Parses input arguments and creates a new AddCommand object
+ */
+public class ScheduleCommandParser implements Parser<ScheduleCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ScheduleCommand
+     * and returns an ScheduleCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ScheduleCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_LOCATION, PREFIX_START_TIME, PREFIX_END_TIME);
```

Comment:
This Parser class looks very neat. Just remember to test inputs with whitespace because I faced some errors due to tokenizing with FindCommand.